README.md

```diff
@@ -39,6 +39,10 @@ To properly use Draft to containerize your application, create manifest files, a
 
 -  Supported deployment types: Helm, Kustomize, Kubernetes manifest.
 
+**Build an image on Azure Container Registry** easily build an image on an ACR through the command palette to use for your deployment files.
```

Comment:
maybe reword to 

"easily build an image on an ACR to later use for your deployment files."

Not sure it's relevant to specify it's through the command palette since they all are.